packages/ia-user-list/src/ia-user-list.ts

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable no-undef */
+/* eslint-disable lit/no-value-attribute */
+
+import { html, css, LitElement } from 'lit';
+import { property, customElement, query } from 'lit-element/decorators.js';
+
+import type { listDataModel } from './models';
+import IAButtonStyles from './style/ia-button';
+
+@customElement('ia-user-list')
+export class IAUserList extends LitElement {
+  /**
+   * contains list information
+   *
+   * @type {UserModel}
+   * @memberof IAUserList
+   */
+  @property({ type: Object }) listData: listDataModel;
+
+  /**
+   * base api URL for user list service
+   *
+   * @memberof IAUserList
+   */
+  @property({ type: String }) apiURL: string;
```

Comment:
```suggestion
  @property({ type: String }) baseHost: string = 'archive.org';
```

We use baseHost everywhere, let's keep using that. + add default if not provided as such.